api/defaults.go

```diff
@@ -411,630 +411,28 @@ func buildSigningPhrase() (string, error) {
 	}
 
 	return dictionary[a.Int64()] + " " + dictionary[b.Int64()] + " " + dictionary[c.Int64()], nil
+}
 
+func randomWalletEmoji() (string, error) {
+	count := big.NewInt(int64(len(animalsAndNatureEmojis)))
+	index, err := rand.Int(rand.Reader, count)
+	if err != nil {
+		return "🐵", err
```

Comment:
Do we need to return an emoji in case of an error?

Reply:
Oops. My intention was to return some emoji and ignore the error.

Just so that no such weird errors don't effect the account creation. And the emoji can always be replaced with anything else later.

Will remove the error argument from here, thanks!

Reply:
Or maybe just return an error and don't overcomplicate 🤔

Reply:
Removed the emoji return. Let's face the error before trying to fix it.
d9c444c7db1dd677ed20f02667bfd39de14a1300